sirenabailie/Fan-Fusion-FE
# Create Story form now working with tags, categories, and targetAudience. routing after create needs debugging

## Detailed Description

Target audience drop-down was fixed fixed, from displaying nothing,  to showing the data and pushing it.
The story form 

## How to Test


1.  Hit the Create Story button in the navbar.
2. Fill out the form.
3. Nothing will appear to happen, but click the "Fan Fusion" in the navbar to go back to home page.
4. The story you created should now be visible.

## Motivation and Context

This solves the problem of creating a story.

## Screenshots (if applicable)

## Types of Changes

- [ x] 🐛 Bug fix
- [ x] ✨ New feature
- [ x] 💡 Improvement
- [ x] 🧹 Code cleanup

## 💬 Additional Notes

Lines 34 to 36 have a throw away function. This is just to call "stories" so the linter wont be mad.
